src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
```

Comment:
Could have header comments for the `Printable` interface (as a whole) as well :)